chia/cmds/init_funcs.py

```diff
@@ -99,7 +120,7 @@ def check_keys(new_root: Path, keychain: Optional[Keychain] = None) -> None:
         updated_target = True
     elif config["farmer"]["xch_target_address"] not in all_targets:
         print(
-            f"WARNING: using a farmer address which we don't have the private"
+            f"WARNING: using a farmer address which we might not have the private"
```

Comment:
Should `f"WARNING: using a pool address which we don't have the private"` also be updated to use the same language?

Reply:
Yes thank you